nilearn/_utils/niimg.py

```diff
@@ -0,0 +1,162 @@
+import copy
+import gc
+import collections
+from six import string_types
+
+import numpy as np
+
+import nibabel
+
+from .numpy_conversions import as_ndarray
+
+
+def _get_shape(img):
```

Comment:
Since SpatialImage should provide `shape` property, I am in favor of removing this one.